packages/wonder-blocks-form/src/__tests__/custom-snapshot.test.tsx

```diff
@@ -5,7 +5,7 @@ import CheckboxCore from "../components/checkbox-core";
 import RadioCore from "../components/radio-core";
 
 const states = ["default", "error", "disabled"];
-const checkedStates = [false, true];
+const checkedStates = [false, true, null || undefined];
```

Comment:
**nit:** This will evaluate to `undefined`, so you can probably use `null` here to be more explicit the indeterminate case:

```suggestion
const checkedStates = [false, true, null];
```

P.S. Don't forget to update the snapshots :).